tests/api/api.spec.ts

```diff
@@ -835,6 +835,18 @@ suite("@dataform/api", () => {
         "select 1 from `tada-analytics.df_integration_test.example_view`"
       );
     });
+    test("bigquery_example with explicit schema", async () => {
+      const compiledQuery = await query.compile(
+        'select 1 from ${ref("df_integration_test", "example_view")}',
+        {
+          projectDir: "examples/common_v1",
+          projectConfigOverride: { warehouse: "bigquery", defaultDatabase: "tada-analytics" }
```

Comment:
```suggestion
          projectConfigOverride: { warehouse: "bigquery", defaultDatabase: "tada-analytics", defaultSchema: "not_a_schema" }
```

I think, otherwise this isn't testing the override actually happens?

Reply:
no override is happening, it's just explicitly referencing a schema in `ref`. 

note that without the bugfix, this test does indeed fail.